clang-tools-extra/docs/clang-tidy/checks/bugprone/pointer-arithmetic-on-polymorphic-object.rst

```diff
@@ -19,20 +19,23 @@ Example:
 .. code-block:: c++
 
   struct Base {
-    virtual void ~Base();
+    virtual ~Base();
+    int i;
   };
-
```

Comment:
Sorry I wasn't clear, I meant to keep the spacing here between structs/functions, but when there is a blank line to not have any whitespace characters on it ('' vs '  '). A couple blanks lines in this first block between structs/functions and LGTM!

Reply:
Do you mean like this?

![image](https://github.com/user-attachments/assets/9ea57dc0-0820-4147-bf08-ec3bb96f704a)

Reply:
So should I make any changes?

Reply:
Yes, I like the blank lines between functions/structs like in your screenshot above. Current PR version has structs and functions with no breathing room. Then good to ship.

Reply:
Check it please.

Reply:
LGTM! Sorry for confusion. Do you need me to click the merge button?